demos/defender/aws_iot_demo_defender.c

```diff
@@ -268,27 +278,29 @@ int RunDefenderDemo( bool awsIotMqttMode,
     if( status == EXIT_SUCCESS )
     {
         /* Create MQTT Connection */
-        mqttStatus = _establishMqttConnection( pIdentifier,
```

Comment:
This function DOES NOT return an mqttStatus. It returns either EXIT_SUCCESS or EXIT_FAILURE.